dist/mjs/interface/interface.d.ts

```diff
@@ -170,11 +170,11 @@ export interface Round2Info {
     _id?: any;
     round_2_ids: string[];
     task_ids: string[];
-    target_instances: number[][];
-    proof: number[];
-    batch_instances: number[];
-    shadow_instances?: number[];
-    aux: number[];
+    target_instances: Uint8Array[];
```

Comment:
Sorry, what I can see in the helper for array normally is Arrary<Uint8Array>?
For example, in Task, we use public_inputs: Array<string>;
In VerifyBatchProofParams, we use sibling_instances: Array<Uint8Array>;
So this one better to be Array<Uint8Array>?